core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
+}
+
 @media (max-width: 768px) {
   .oppia-footer-padding {
     height: 464px;
   }
+
+  .oppia-footer-margin {
```

Comment:
Generally speaking, simpler is better unless it has unintended consequences. I defer to @charisseysabel  on this.